### DISCUSSION TITLE
Exploring storage options

### DISCUSSION THREAD

**Discussion by sanjeevi55** [outline/outline#5108] [General]
Hi,

I would like to store the files locally on a self-hosted server instead of using S3, minio or any other tool. I have a reason for this. Is it possible to do that? If yes, could anyone help with how to do that? 

Thanks in advance.

**Comment by helmut72** [comment]
A local storage would be really cool!

I only need MinIO because of Outline. But MinIO melts any SSD like icecream in the summer, even if it's idling most of the time. See this:
https://github.com/minio/minio/discussions/15705

I only have a single instance of MinIO running, no need to write to my SSD especially when idling.

Lightweight S3 alternatives like [GarageHQ](https://garagehq.deuxfleurs.fr/) doesn't work with Outline, because GarageHQ doesn't support anonymous access which is required by Outline:
https://git.deuxfleurs.fr/Deuxfleurs/garage/issues/263

Result: Most of the time Outline isn't up and I'm glad it's for now for testing only.
